storage/container_storage_interface/persistent-storage-csi-migration.adoc

```diff
@@ -0,0 +1,30 @@
+[id="persistent-storage-csi-migration"]
+= CSI automatic migration
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-migration
+
+toc::[]
+
+== Overview
```

Comment:
Usually we don't add an "Overview" heading before the introductory assembly information. I would either remove this heading, or move this information into a module and include that (but still add at least 1 intro sentence in the assembly first).